packages/aws-cdk-lib/aws-ec2/lib/machine-image/machine-image.ts

```diff
@@ -176,7 +191,7 @@ export abstract class MachineImage {
 }
 
 /**
- * Select the image based on a given SSM parameter
+ * Select the image based on a given SSM parameter at deploy time.
```

Comment:
Nice! But maybe we can make this even more clear:
```suggestion
 * Select the image based on a given SSM parameter at deployment time of the CloudFormation Stack.
```